pkg/scheduler/factory/factory.go

```diff
@@ -670,10 +670,8 @@ func (c *configFactory) updatePodInCache(oldObj, newObj interface{}) {
 			case schedulerapi.ResizeActionReschedule:
 				// Case 2. Node does not have capacity. Delete pod, let controller re-create pod.
 				c.recorder.Eventf(newPod, v1.EventTypeNormal, "DeletePodForResizeReschedule", "Deleting pod %s to reschedule for resizing", podName)
-				deleteOptions := metav1.NewDeleteOptions(0)
-				deleteOptions.Preconditions = metav1.NewUIDPreconditions(string(newPod.UID))
```

Comment:
why removing these options? (or what do these options actually do?)

Reply:
i noticed that the delete was returning error, though the delete was actually working as expected.